Pipfile

```diff
@@ -17,13 +17,12 @@ requests = "==2.*"
 tzwhere = "*"
 babel = "*"
 pybreaker = "*"
-osm-humanized-opening-hours = "==0.6.2"
 redis = "*"
 python-redis-rate-limit = "*"
-lark-parser = "<0.6"
 prometheus-client = "*"
 apistar-prometheus = "*"
 python-json-logger = "*"
+osm-humanized-opening-hours = {editable = true, ref = "2fde45bec", git = "https://github.com/rezemika/humanized_opening_hours.git"}
```

Comment:
We can directly use `osm-humanized-opening-hours==1.0.0b2` now that it has been released